staging/src/k8s.io/code-generator/go.mod

```diff
@@ -6,6 +6,19 @@ go 1.24.0
 
 godebug default=go1.24
 
+tool (
```

Comment:
do we know if these get propagated to things which `_` import `k8s.io/code-generator`? I'm a little fuzzy on what this bit will do to things downstream of code-generator